### PR TITLE
Videomaker: Fix category trailing slash 

### LINKED ISSUE #4882: Videomaker: Trailing slash when no category set
## Steps to replicate

When posts don't have a category, the slash between category and date looks kind of weird:
Screenshot 2021-10-20 at 20 41 29

Any thoughts on this one @beafialho

### PR DESCRIPTION
**Pull request by mikachan**
Co-authored-by: @danieldudzic

#### Changes proposed in this Pull Request:
This PR fixes the trailing slash if a post does not have categories.

image

This also removes the tag styles if tags are added to the `post-meta` group:

image

#### Related issue(s):
Closes #4882

### CODE REVIEW

File: videomaker/sass/_meta.scss
Comment (MaggieCabrera) [inline videomaker/sass/_meta.scss]: Do we need this because you are using before for the slash? or do we need this anyway? Beacause I'm thinking, a different way of doing this would be something like: `.post-meta p:last-child { display: none; }` it will only happen when the post-terms blocks is not present because afaik when it's not there there is no empty div.

Comment (mikachan) [reply videomaker/sass/_meta.scss]: Oh yeah that sounds good! All this was because I was using the before for the slash. Let me try this out now.

Comment (mikachan) [reply videomaker/sass/_meta.scss]: Ah, it was also added to handle multiple post meta blocks being added by the user (or the user just moving these blocks around in general). At the moment, I think users would need to manually add a slash themselves between each item.

Comment (MaggieCabrera) [reply videomaker/sass/_meta.scss]: I'm going to start a PR to move the icons outside of the general post meta styles

Comment (MaggieCabrera) [reply videomaker/sass/_meta.scss]: [This](https://github.com/Automattic/themes/pull/4891) removes all the extra CSS from the icons if the class is not applied, should reduce the needed CSS here

Comment (MaggieCabrera) [reply videomaker/sass/_meta.scss]: Also, Quadrat does the same thing with a dot as a separator, so maybe you want to grab inspiration from that.

Comment (mikachan) [reply videomaker/sass/_meta.scss]: I've updated this so Videomaker is tweaking the styles in the same way as Quadrat. I think it looks good, the styles definitely look better!

This means that this change needs [your PR](https://github.com/Automattic/themes/pull/4891) before it's merged.

Comment (MaggieCabrera) [reply videomaker/sass/_meta.scss]: Just did! now I think this needs a rebase

Comment (mikachan) [reply videomaker/sass/_meta.scss]: Woop, this is now up to date with trunk and still working 😁

File: videomaker/sass/_meta.scss
Comment (scruffian) [inline videomaker/sass/_meta.scss:11]: what happens with i18n and RTL if we take this approach? I guess we had that problem before :)

File: videomaker/block-template-parts/post-meta.html
Comment (scruffian) [inline videomaker/block-template-parts/post-meta.html]: Should we use the block-gap variable here?

Comment (mikachan) [reply videomaker/block-template-parts/post-meta.html]: Yeah good idea! I think variables usually make more sense than fixed values.

**Comment by scruffian** [comment]
In RTL this did look like this
Screenshot 2021-10-22 at 09 55 41

With the changes I just made it now looks like this:
Screenshot 2021-10-22 at 09 55 52

**Comment by scruffian** [comment]
@mikachan I updated this to use the --wp--style--block-gap variable as that's what we use for spaces between blocks. I also added 1em as a fallback.

**Review by scruffian — APPROVED** [review]
LGTM if you're happy with my changes

**Comment by mikachan** [comment]
> @mikachan I updated this to use the --wp--style--block-gap variable as that's what we use for spaces between blocks. I also added 1em as a fallback.

Ah thanks, I didn't realise. These changes look good to me, thanks for your help!

**Pull request merged**
